libexec/bats-core/bats-exec-test

```diff
@@ -317,18 +319,17 @@ bats_perform_test() {
   # function when the ERR trap fires. All versions of Bash appear to reset it
   # on an unbound variable access error. bats_debug_trap will fire both before
   # the offending line is executed, and when the error is triggered.
-  # Consequently, we use `BATS_LINENO` to point to the line number seen by the
+  # Consequently, we use `BATS_CURRENT_STACK_TRACE` recorded by the
   # first call to bats_debug_trap, _before_ the ERR trap or unbound variable
   # access fires.
-  BATS_CURRENT_LINENO=0
-  BATS_LINENO=0
   BATS_STACK_TRACE=()
+  BATS_CURRENT_STACK_TRACE=()
 
   BATS_TEST_COMPLETED=
   BATS_TEST_SKIPPED=
   BATS_TEARDOWN_COMPLETED=
   BATS_ERROR_STATUS=
-  trap "bats_debug_trap \"\$BASH_SOURCE\"" debug
+  trap 'bats_debug_trap "$BASH_SOURCE"' debug
```

Comment:
This change results in the value `BASH_SOURCE` being printed as is.

Reply:
No, it's the same as before just less escapes and hence less confusing

Reply:
Oh you are right and I am an idiot :D